css/splitbox.css

```diff
@@ -0,0 +1,33 @@
+.splitbox {
```

Comment:
Yes, maybe you want to use it on a project and you don't have the sass to css pipeline setup. It should be fairly simple to get started with the component.